README.rst

```diff
@@ -13,1837 +13,30 @@ Turn complex GraphQL queries into optimized database queries.
 Quick Overview
 --------------
 
-Through the GraphQL compiler, users can write powerful queries that
-uncover deep relationships in the data while not having to worry about
-the underlying database query language. The GraphQL compiler turns
-read-only queries written in GraphQL syntax to different query
-languages.
+The GraphQL Compiler is a library that simplifies database querying and exploration by exposing one
```

Comment:
```suggestion
The GraphQL compiler is a library that simplifies database querying and exploration by exposing one
```
Not sure if this should be capitalized. I don't believe that we've capitalized it historically in blog posts or documentation. @JulianGoetz what do you think?

Reply:
I'm seeing this without "The" in the blog posts. I remember asking when you were writing them why we don't have "the" and you made a compelling case for not having it, though I don't remember now what that was. Otherwise, we haven't been capitalizing "compiler". https://blog.kensho.com/database-agnostic-querying-is-unavoidable-at-scale-18895f6df2f0

Reply:
Removed the capitalization from Compiler and removed the "The" prefix.